notebooks/COS/DataDl/DataDl.ipynb

```diff
@@ -160,7 +160,7 @@
     "\n",
```

Comment:
<p>Similar to the one at the top, including an enumeration for the loop in the code is unnecessary. Instead, we could achieve the same result with the following:</p><p> </p><pre class="ql-syntax" spellcheck="false">sourcelist = ['omega Centauri', 'M5', 'M13', 'M15', 'M53']
with open('./objectname_list.csv', 'w') as f:
    for source in sourcelist[:-1]:
        f.write(source + "," )
&nbsp;&nbsp;  f.write(sourcelist[-1])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Thanks for updating the code. No need for these lines either:</p><p> </p><pre class="ql-syntax" spellcheck="false"># Measures the length of the list for if statements below
sourcelist_length = len(sourcelist)
</pre>